examples/pytorch/dagnn/README.md

```diff
@@ -0,0 +1,34 @@
+# DAGNN
+
+### Performance
+
+#### On Cora, Citeseer and Pubmed
+##### Accuracy
+| Dataset | Cora | Citeseer | Pubmed |
+| :-: | :-: | :-: | :-: |
+| DAGNN | 84.4 ± 0.5 | 73.3 ± 0.6 | 80.5 ± 0.5 |
+| DGL | 84.3 ± 0.6 | 73.5 ± 0.8 | 80.5 ± 0.3 |
+
+#### On Arxiv
```

Comment:
Is this ogbn-arxiv? If so, I guess it's better to use that term. Also you can directly merge the two tables.

Reply:
Now I only leave citation datasets' results.